JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -112,15 +139,15 @@ private fun PublicationsTab() {
 @Composable
 private fun TabWithTopics(tabname: String, topics: List<String>) {
     VerticalScroller {
-        Column {
-            Spacer(LayoutHeight(16.dp))
+        Column(modifier = LayoutPadding(top = 16.dp)) {
             topics.forEach { topic ->
                 TopicItem(
                     getTopicKey(
                         tabname,
                         "- ",
                         topic
-                    ), topic
+                    ),
+                        topic
```

Comment:
Weird formatting

Reply:
Have I mentioned how much I love the default formatting in AS :). Updating manually.